os/arch/arm/src/amebad/Kconfig

```diff
@@ -12,493 +12,50 @@ choice
-		default n
-
-	config CRYPTO_VERIFY
-		bool "Crypto_Verify_Mode"
-		default n
-
-	config EFUSE_VERIFY
-		bool "EFUSE_Verify_Mode"
-		default n
-
-	config SPIC_VERIFY
-		bool "SPIC_Verify_Mode"
-		default n
-
-	config FTL_VERIFY
-		bool "FTL_Verify_Mode"
-		default n
-
-	config ADC_VERIFY
-		bool "ADC_Verify_Mode"
-		default n
-
-	config RSIP_VERIFY
-		bool "RSIP_Verify_Mode"
-		default n
-
-	config SDIO_DEVICE_VERIFY
-		bool "SDIO_DEVICE_Verify_Mode"
-		default n
-
-	config SDIOH_VERIFY
-		bool "SDIO_HOST_Verify_Mode"
-		default n
-
-	config DHRYSTONE_TEST
-		bool "Dhrystone Test"
-		default n
-
-	config RTLIB_VERIFY
-		bool "RTLIB ROM Test"
-		default n
-
-	config IPC_VERIFY
-		bool "IPC Test"
-		default n
-
-	config IR_VERIFY
-		bool "IR Test"
-		default n
-
-	config MPU_VERIFY
-		bool "MPU Test"
-		default n
-
-	config CACHE_VERIFY
-		bool "CACHE Test"
-		default n
-
-	config WFE_VERIFY
-		bool "WFE Test"
-		default n
-
-endif
-
-menuconfig SEC_VERIFY
-	bool "SEC_Verify_Mode"
+config RTL8721D_UART0
+	bool "UART 0"
 	default n
+	select RTL8721D_UART
+	select ARCH_HAVE_UART0
+	select ARCH_HAVE_SERIAL_TERMIOS
 
-if SEC_VERIFY
-
-	config CACHE_SEC_VERIFY
-		bool "CACHE Secure Test"
-		default n
-
-	config EFUSE_SEC_VERIFY
-		bool "EFUSE Secure Test"
-		default n
-
-	config GDMA_SEC_VERIFY
-		bool "GDMA Secure Test"
-		default n
-
-	config AES_SEC_VERIFY
-		bool "AES Secure Test"
-		default n
-
-	config CRYPTO_SEC_VERIFY
-		bool "Crypto Secure Test"
-		default n
-
-	config CRC_SEC_VERIFY
-		bool "CRC Secure Test"
-		default n
-
-	config RDP_SEC_VERIFY
-		bool "RDP Secure Test"
-		default n
-endif
-
-
-config BT_EN
-	bool "Enable BT"
+config RTL8721D_UART1
+	bool "UART 1"
 	default n
+	select RTL8721D_UART
+	select ARCH_HAVE_UART1
+	select ARCH_HAVE_SERIAL_TERMIOS	
 
-if BT_EN
-
-	config BT
-		bool
-		default n
-
-	config BT_PERIPHERAL
-		bool "BT_Peripheral"
-		default n
-
-	config BT_CENTRAL
-		bool "BT_Central"
-		default n
-
-	config BT_SCATTERNET
-		bool "BT_Scatternet"
-		default n
-		depends on BT_PERIPHERAL && BT_CENTRAL
-
-	config BT_BEACON
-		bool "BT_Beacon"
-		default n
-
-	config BT_CONFIG
-		bool "BT_Simple_Config"
-		default n
-
-	config BT_AIRSYNC_CONFIG
-		bool "BT_Airsync_Config"
-		default n
-endif
-
-
-menuconfig WIFI_EN
-	bool "Enable Wifi"
+config RTL8721D_UART2
+	bool "UART 2"
 	default n
-	select WIFI_NORMAL
-
-if WIFI_EN
-
-	config HIGH_TP_TEST
-		bool "Enable High TP Test"
-		default n
-
-choice
-	prompt "WIFI Config Type"
-	default WIFI_NORMAL
-
-	config WIFI_NORMAL
-		bool "Normal Wi-Fi"
-		select WIFI_MODULE
-
-	config WIFI_TEST
-		bool "Test Wi-Fi"
-		select WIFI_MODULE
-		select WIFI_VERIFY
-
-endchoice
-
-	config WIFI_MODULE
-		bool
-		default n
-		depends on WIFI_NORMAL
-
-	config WIFI_VERIFY
-		bool
-		default n
-		depends on WIFI_TEST
-
-endif
+	select RTL8721D_UART
+	select ARCH_HAVE_UART2
+	select ARCH_HAVE_SERIAL_TERMIOS
 
-
-config NETWORK
-	bool "Enable Network"
+config CONFIG_RTL8721D_SERIAL_FIFO
```

Comment:
Could you remove `CONFIG_`? Because of that, it becomes `CONFIG_CONFIG_RTL8721D_SERIAL_FIFO`.

Reply:
I will resolve this yes, it is an issue.

Reply:
fixed in 
a87cb70